### DISCUSSION TITLE
Authentication failed against database server at `localhost`

### DISCUSSION THREAD

**Discussion by menglingyu** [prisma/prisma#6286] [Q&A]
mysql -u root -p can connection mysql but prisma cant

 Error: Authentication failed against database server at `localhost`, the provided database credentials for `root` are not valid.

```
"prisma/schema.prisma" 24L, 493C                                                                                                                                                                  3,3          全部
```
```prisma
datasource db {
  provider = "mysql"
  url      = "mysql://root:123456:@localhost:3306/zhihu"
}

generator client {
  provider = "prisma-client-js"
}

model Post {
  id        Int     @id @default(autoincrement())
  title     String
  content   String?
  published Boolean @default(false)
  author    User?   @relation(fields: [authorId], references: [id])
  authorId  Int?
}

model User {
  id    Int     @id @default(autoincrement())
  email String  @unique
  name  String?
  posts Post[]
}
```

mysql  password is 123456

**Comment by ryands17** [comment]
@menglingyu 👋 

It would be great if you could open a bug report [here](https://github.com/prisma/prisma/issues/new?assignees=&labels=&template=bug_report.md&title=) regarding the same with the details.

**Comment by janpio** [comment]
If your password is `123456`, your connection string has an additional `:` at the end that you should remove: `123456:`.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
